exercises/java/encryption-context-complete/src/main/java/sfw/example/esdkworkshop/Api.java

```diff
@@ -54,7 +55,16 @@ public Api(
       AmazonS3 s3Client,
       String bucketName,
       MasterKeyProvider mkp) {
-    this(ddbClient, tableName, s3Client, bucketName, new AwsCrypto(), mkp);
+    this(
+        ddbClient,
+        tableName,
+        s3Client,
+        bucketName,
+        AwsCrypto.builder()
+            .withCommitmentPolicy(CommitmentPolicy.RequireEncryptRequireDecrypt)
+            .build(),
```

Comment:
nit: Can we just use `AwsCrypto.standard()` here? I don't think it's worth the distraction to pull in the commitment policy concept here.

Reply:
Never mind, `AwsCrypto.standard()` isn't in the 1.x ESDK versions. But then neither is this commitment policy, we have to stick to `ForbidEncryptAllowDecrypt` to work on both major versions.

Reply:
Per offline discussion, since we want this to work with 1.x and 2.x, we can't use `standard`, but we also can't use the existing 2.0-only commitment policy. I've updated it to use the default commitment policy, which will work with both versions.